neurodamus/core/configuration.py

```diff
@@ -877,10 +877,11 @@ def _coreneuron_params(config: _SimConfig, run_conf):
     coreneuron_datadir = os.path.join(config.output_root, "coreneuron_input")
 
     if config.use_coreneuron and config.restore:
-        # Most likely we will need to reuse coreneuron_input from first part
+        # Most likely we will need to reuse coreneuron_input from the save part
+        # A symlink is created for the scenario of multiple save/restore processes in one simulation
         if not os.path.isdir(coreneuron_datadir):
-            logging.info("RESTORE: Searching for coreneuron_input besides " + config.restore)
-            coreneuron_datadir = os.path.join(config.restore, "..", "coreneuron_input")
+            logging.info("RESTORE: link to coreneuron_input besides " + config.restore)
```

Comment:
This message is a bit confusing i feel.
Maybe something like
`logging.info(f"RESTORE: Creating symlink for coreneuron_input from {config.restore}")`

Reply:
Yes, sure